services/crypto/key/threats.yaml

```diff
@@ -1,18 +1,32 @@
 common_threats:
   - CCC.TH01
+  - CCC.TH03
+  - CCC.TH13
+
```

Comment:
We need CCC.TH05 here as CCC.C06 is added under common_controls for KMS and CCC.C06 refers to CCC.TH05

Reply:
Add all the referred threats need their title as a comment in front.
Eg - CCC.TH01 # Unauthorized access through elevated privileges